component-samples/coding-components/samples/react/src/index.tsx

```diff
@@ -17,12 +17,16 @@ import React from "react";
 import ReactDOM from "react-dom/client";
 
 import ArcadeEditor from "./components/ArcadeEditor";
-import { defineCustomElements as defineCalciteElements } from "@esri/calcite-components/dist/loader";
-import { defineCustomElements as defineCodingElements } from "@arcgis/coding-components/dist/loader";
 
-// define custom elements in the browser, and load the assets from the CDN
-defineCodingElements(window, { resourcesUrl: "https://js.arcgis.com/coding-components/4.30/assets" });
-defineCalciteElements(window, { resourcesUrl: "https://js.arcgis.com/calcite-components/2.8.5/assets" });
+// Individual imports for each component
+import { setArcgisAssetPath as setCodingComponentsAssetPath } from "@arcgis/coding-components/dist/components";
+import "@arcgis/coding-components/dist/components/arcgis-arcade-editor";
+
+import { setAssetPath as setCalciteComponentsAssetPath } from '@esri/calcite-components/dist/components';
+import "@esri/calcite-components/dist/components/calcite-scrim";
+
+setCodingComponentsAssetPath("https://js.arcgis.com/coding-components/4.30/assets");
+setCalciteComponentsAssetPath("https://js.arcgis.com/calcite-components/2.8.0/assets");
```

Comment:
```suggestion
// Individual imports for each component
import "@arcgis/coding-components/dist/components/arcgis-arcade-editor";
import "@esri/calcite-components/dist/components/calcite-scrim";

import { ArcgisArcadeEditor } from "@arcgis/coding-components-react";
import { CalciteScrim } from "@esri/calcite-components-react";
```

You'll have to update `react/src/components/ArcadeEditor.tsx`

You don't need to set an asset path when individually importing components, we pick up CDN hosted assets by default.